client/my-sites/patterns/components/get-started/index.tsx

```diff
@@ -99,10 +100,16 @@ export function PatternsGetStarted() {
 					/>
 					<div className="patterns-get-started__item-name">{ translate( 'Free course' ) }</div>
 					<div className="patterns-get-started__item-description">
-						{ translate( 'Design Your Homepage', {
-							comment:
-								'This string is a copy of the page title from wordpress.com/learn/webinars/compelling-homepages/',
-						} ) }
+						{ hasTranslation( 'Launch your site faster' ) &&
```

Comment:
Are you sure that we should ignore the comment in `hasTranslation`?

Reply:
@nightnei I've checked with i18n team, only the string+context (not comment) matter when determining the key for showing the translations. Given that we didn't supply any `context` to `translate()`, we should be good.